seed/views/v3/organizations.py

```diff
@@ -1510,7 +1510,7 @@ def insert_sample_data(self, request, pk=None):
         import_record = ImportRecord.objects.create(name='Auto-Populate', super_organization=org)
 
         # Interval Data
-        filename = 'PM Meter Data 12.xlsx'
+        filename = 'PM Meter Data.xlsx'  # contians meter data for BSync and BETTER
```

Comment:
Could you delete the old file?

```suggestion
        filename = 'PM Meter Data.xlsx'  # contians meter data for bsyncr and BETTER
```